raiden/tests/integration/api/test_pythonapi.py

```diff
@@ -101,6 +104,26 @@ def test_raidenapi_channel_lifecycle(raiden_network, token_addresses, deposit, r
             registry_address=registry_address, token_address=None, partner_address=api2.address
         )
 
+    # Make sure a small settle timeout is not accepted when opening a channel
+    with pytest.raises(InvalidSettleTimeout):
+        invalid_settle_timeout = node1.raiden.config["reveal_timeout"] * 2 - 1
+        api1.channel_open(
+            registry_address=node1.raiden.default_registry.address,
+            token_address=token_address,
+            partner_address=api2.address,
+            settle_timeout=invalid_settle_timeout,
+        )
+
+    # Make sure a too large settle timeout is not accepted when opening a channel
+    with pytest.raises(InvalidSettleTimeout):
+        invalid_settle_timeout = settle_timeout_max + 1
+        api1.channel_open(
+            registry_address=node1.raiden.default_registry.address,
+            token_address=token_address,
+            partner_address=api2.address,
+            settle_timeout=invalid_settle_timeout,
+        )
```

Comment:
Actually, I was suggesting to test that `channel_open` will succeed for `settle_timeout = node1.raiden.config["reveal_timeout"] * 2` to verify that the settle_timeout validity threshold is honored in both directions.

But maybe the condition is so simple that a test for that is overkill. Add it or leave it at your own discretion.

Reply:
done